File: harshil/Day_02.java
// Q-link: https://www.codechef.com/problems/BUDGET25

import java.util.*;
import java.lang.*;
import java.io.*;

class Codechef {
    public static void main(String[] args) throws java.lang.Exception {
        // your code goes here
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        while (t-- > 0) {
            int n = sc.nextInt();
            int x = sc.nextInt();
            ArrayList<Integer> arr = new ArrayList();
            long count = 0;
            long surplus = 0;
            while (n-- > 0) {
                int a = sc.nextInt();
                if (a >= x) {
                    count++;
                    surplus += a - x;
                } else {
                    arr.add(x - a);
                }
            }
            Collections.sort(arr);
            for (int el : arr) {
                if (surplus >= el) {
                    count++;
                    surplus -= el;
                } else
                    break;
            }
            System.out.println(count);
        }

    }
}

File: raj/Day_02.py
# Question -> https://www.codechef.com/problems/BUDGET25?tab=statement
# cook your dish here
for i in range(int(input())):
    n, x = map(int, input().split())
    ln = list(map(int, input().split()))
    
    c = 0
    surplus = 0
    for i in range(n):
        if ln[i] > x:
            diff = ln[i] - x
            ln[i] -= diff
            surplus += diff

    ln.sort()                                
    
    for i in range(n-1, -1, -1):
        if ln[i] < x and surplus >= x - ln[i]:
            don = x - ln[i]
            ln[i] += don
            surplus -= don
    
    for i in range(n):
        if ln[i] >= x:
            c += 1
    
    print(c)
